docs/codeql/codeql-cli/about-codql-packs.rst

```diff
@@ -0,0 +1,101 @@
+.. _about-codeql-packs:
```

Comment:
Driveby comment.

There's a typo in the file name.

Reply:
Good catch. ✨